### DISCUSSION TITLE
Automated Host for TripleA 2.6

### DISCUSSION THREAD

**Discussion by Cernelius** [triplea-game/triplea#12192] [General]
If the release is closing, I assume it is advisable already having one automated host in the new lobby for testing.

(I'm not a bot user myself, but many are.)

**Comment by Cernelius** [comment]
Also, you may want to decide whether or not to reduce the number of TripleA automated hosts in order to save money. I reported that 2 or 3 of the 4 "California" ones are often unavailable (broken) in the current TripleA 2.5 lobby.

**Reply by DanVanAtta** [reply]
Good point, sounds like the California bot should be closed down. (Finding time and opportunity to do so might be a challenge, volunteers would be helpful here)

**Comment by DanVanAtta** [comment]
> If the release is closing, I assume it is advisable already having one automated host in the new lobby for testing.

Hard to say, hopefully.. I think the suggestion makes sense. I've a rough plan with an order of operations. By the time we can add those bots, it might be 'release' time. I understand though that bots connected to the new lobby would be of benefit, if I can move that up; I'll try to do so.
